apps/astarte_pairing/lib/astarte_pairing_web/router.ex

```diff
@@ -38,6 +38,11 @@ defmodule Astarte.PairingWeb.Router do
 
     get "/version", VersionController, :show
 
+    scope "/ownership-voucher" do
```

Comment:
While it's not always the case, usually Astarte APIs have `_` as word separator

```suggestion
    scope "/ownership_voucher" do
```

Reply:
I was the one suggesting the `-`, as the only multi-word route (excluding parameter names) I could find was `/v1/realm/devices-by-alias/` in appengine. Where are we using `_` in routes?